_topic_map.yml

```diff
@@ -1065,6 +1065,8 @@ Topics:
     File: persistent-storage-csi-cloning
   - Name: AWS Elastic Block Store CSI Driver Operator
     File: persistent-storage-csi-ebs
+  - Name: Azure CSI Driver Operator
```

Comment:
This should be renamed "Azure **Disk** CSI Driver Operator" to distinguish from [Azure **File** CSI](https://issues.redhat.com/browse/STOR-554), which this feature does not cover.